### ISSUE TITLE
Dashboard filters should apply aria-expanded attributes on the trigger elements when popovers are open

### ISSUE THREAD

**Issue by iethree** [metabase/metabase#70543]
currently, the popover itself carries the aria-expanded attribute. it should be on the trigger element
